blibli-backend-framework-newrelic/README.md

```diff
@@ -0,0 +1,51 @@
+# Reactor New Relic Instrumentation
+
+A plugin to enable new relic instrumentation in Spring Boot 2 Webflux App.
+
+List of supported modules:
+- command-plugin: anything that implement `Command` interface.
+- spring-reactive-mongo: ReactiveMongoRepository all inherited method and custom query.
+
+Planned future modules:
+- Reactive relational DB (via R2DBC)
+- Reactive elastic search
+- Reactive redis
+
+## How to use
+
+1. Include the library in your POM.
+
+    ```
+    <dependency>
+        <groupId>com.blibli.oss</groupId>
+        <artifactId>blibli-backend-framework-newrelic</artifactId>
+    </dependency>
+    ```
+
+2. Add `@EnableNewRelicReactorInstrumentation` annotation to enable instrumentation in the main app.
+    
+    ```java
+    @SpringBootApplication
+    @EnableNewRelicReactorInstrumentation
```

Comment:
can we remove this? just automatically register configuration on spring, don't manually register using annotation

Reply:
deleted already and set as auto configured now